src/pages/Components/Radio/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Radio'
+order: 8
+---
+
+> Checkboxes are used to **select only one option in a list**. They usually find their place in forms.
```

Comment:
Radio, not Checkboxes

Reply:
To complete the definition of a radio :
Radios are components that, unlike checkboxes, can't be used alone.
They are used when you have to make a single choice in a list, that includes minimum two options.

Reply:
"Used to select a single option from a list". I think it's saying the same thing in fewer words... 
I think the "They usually find their place in forms." is unnecessary.